programs/ziti-edge-tunnel/config-utils.c

```diff
@@ -138,3 +138,28 @@ void update_identity_config(uv_loop_t *l, const char *identifier, const char *cf
     }
 }
 
+char* resolve_directory(const char* path) {
+    char* resolved_path = (char*)malloc(PATH_MAX);
+    if (access(path, F_OK) != -1) {
+        //means the file exists right where it is, use realpath and normalize it and continue
+        if (realpath(path, resolved_path) == NULL) {
+            //how could we get here?
+            printf("path does not exist or permission denied: %s\n", resolved_path);
+            exit(1);
+        }
+    } else {
```

Comment:
Is this needed? Resolving it won't fix access

Reply:
nope. thanks for noticing. will remove